README.md

```diff
@@ -44,12 +66,24 @@ A digital datasheet shall include the following information:
 - The component part type. See appendix for examples.
 - List of Component Pins as defined by the specification.
 
-#### 3.3 Date Format
+#### 4.3 Date Format
 A digital datasheet shall follow the international standard notation, YYYY-MM-DD.
 
-### 3.4	 Common
+####  4.4	 Top Level Component Specification 
+
+Source: [component.json](https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/component.json)
+
```

Comment:
The tool didn't update the links to github.io.

Reply:
whoops - fixed. thanks